kolibri/core/content/api.py

```diff
@@ -1391,12 +1391,7 @@ class UserContentNodeViewset(BaseContentNodeMixin, BaseValuesViewset, ListModelM
 
     def get_queryset(self):
         user = self.request.user
-        # if user is anonymous, don't return any nodes
-        # if person requesting is not the data they are requesting for, also return no nodes
         queryset = models.ContentNode.objects.filter(available=True)
-        if not user.is_facility_user:
```

Comment:
These lines:
```
if not user.is_facility_user:
    user = None
```
Here and below should have been left in, it was only the comment lines that needed to be deleted.


This is causing the failing tests that @bjester highlighted.